code/datums/components/bombable_turf.dm

```diff
@@ -0,0 +1,43 @@
+/**
+ * Apply this to a turf (usually a wall) and it will be destroyed instantly by any explosion.
+ * Most walls can already be destroyed by explosions so this is largely for usually indestructible ones.
+ * For applying it in a map editor, use /obj/effect/mapping_helpers/bombable_wall
+ */
+/datum/component/bombable_turf
+	/// Overlay we show to let you know it can be blown up
+	var/mutable_appearance/overlay
```

Comment:
If the overlays could be added during as a signal proc for whatever signal update_overlays sends, then we could easily make this into an element instead.